examples/hooks/components/Refresh.css

```
@@ -0,0 +1,27 @@
+.Refresh {
+  height: 100%;
+  padding: 0 1rem;
+  margin: 0;
+  font-family: inherit;
+  font-size: 0.875rem;
+  color: #23263b;
+  background: linear-gradient(-180deg, #fff, #fcfcfd);
+  border: 1px solid #d6d6e7;
+  border-radius: 3px;
+  box-shadow: 0 1px 0 0 rgb(35 38 59 / 5%);
+  text-align: center;
+  line-height: 2rem;
+  cursor: pointer;
+}
+
+.Refresh:hover {
+  background: linear-gradient(-180deg, #fff, #f5f5fa);
+}
+
+.Refresh svg {
+  transition: transform .5s ease;
```

This has a smoother ending.

```suggestion
  transition: transform .5s ease-out;
```